examples/logs-basic/Cargo.toml

```diff
@@ -12,4 +12,3 @@ opentelemetry-stdout = { path = "../../opentelemetry-stdout", features = ["logs"
 opentelemetry-appender-log = { path = "../../opentelemetry-appender-log", default-features = false}
 opentelemetry-semantic-conventions = { path = "../../opentelemetry-semantic-conventions" }
 log = { workspace = true }
-serde_json = { workspace = true }
```

Comment:
`serde_json` was added to support this : https://github.com/open-telemetry/opentelemetry-rust/blob/main/examples/logs-basic/src/main.rs#L11-L13 
Could you keep them to reduce the scope of this PR? 

(I am open to revisiting the `serde_json` dep separately.)

Reply:
Sorry, sure. I just wrote `cargo machete` to catch anything that might be dead dependencies after the upgrade, seems like I hit a few things I shouldn't have 🙃